test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -194,24 +206,7 @@ func Test_ValidateNamespace_NoParams(t *testing.T) {
 				return
 			}
```

Comment:
Wondering if this check can also be put into `checkExpectedError` with an early return

Reply:
`checkExpectedError` now early exits if `err` is `nil` and no error was expected - fixed!